__tests__/app.test.js

```diff
@@ -51,7 +51,47 @@ describe("app", () => {
         .expect(200)
         .then(({ body }) => {
           expect(body.endpoints).toEqual(endpoints);
-          expect(typeof body.endpoints["GET /api/articles"].description).toBe("string");
+          expect(typeof body.endpoints["GET /api"].description).toBe("string");
+        });
+    });
+  });
+
+  describe.only("/api/articles/:article_id", () => {
+    test("GET: 200 - should return an array of article objects with the relevant objects keys and value data types", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          const article = body.article;
+          console.log(article);
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.body).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
```

Comment:
It's OK to check the shape of the object, but as we're requesting a certain article, it's better to be more specific and check the article is the exact one we expect